src/ui/theme/components/button.ts

```diff
@@ -8,6 +8,7 @@ const ButtonStyle = {
   // styles for different visual variants ("outline", "solid")
   variants: {
     solid: variantSolid,
+    setting: variantSetting,
```

Comment:
I would rename to toggle variant actually. Now that I think about it more, that seems to make more sense